src/SignalR/clients/csharp/Client.Core/src/IRetryPolicy.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.SignalR.Client
+{
+    public interface IRetryPolicy
```

Comment:
I still think a build would be nice for configuration here but it can wait 
https://github.com/aspnet/AspNetCore/issues/9016

Reply:
If we start adding more options like retrying on initial start failures, this starts to become even more appealing.